src/sort.js

```diff
@@ -1,24 +1,26 @@
 /* tslint:disable:no-console */
 import fs from "fs"
-import glob from "glob"
-import sortJson from "sort-json"
 
 import getLocalesPaths from "./getLocalesPaths"
 
 const modifiedFiles = []
-const paths =  getLocalesPaths()
+const paths = getLocalesPaths()
 
 export default (args) => {
   const fix = args.includes("--fix")
 
   paths.forEach(path => {
     const previousContent = fs.readFileSync(path, "utf8")
-    sortJson.overwrite(path)
-    const newContent = fs.readFileSync(path, "utf8")
+    const newContent = JSON.stringify(JSON.parse(previousContent), null, "  ")
```

Comment:
reading the file with `JSON.parse` results in a sorted json, which is the same as phraseapp. that's probably what they're doing

Reply:
This might be inconsistent even tho JSON parse returns the object sorted according to your comment the stringify  operator is not guaranteed to keep the sort order:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify#Issue_with_plain_JSON.stringify_for_use_as_JavaScript

![Screenshot 2020-02-28 at 15 20 17](https://user-images.githubusercontent.com/3371760/75556373-66f49a80-5a3e-11ea-85b5-75805e4cc585.png)

I don't have another approach so it's fine to me to go with this version and see if we encounter any errors

Reply:
I had that in mind too. If it's not in the specs, it can differ per environment, but I would expect it to be consistent per node-version.